Sources/FoundationExtensions/String+Extensions.swift

```diff
@@ -59,6 +59,28 @@ extension String {
         }
     }
 
+    var asRedactedAPIKey: String {
+        guard let underscoreIndex = self.firstIndex(of: "_") else {
+            return self // No underscore → do not redact
```

Comment:
Hmm I guess this will not redact the very old  API keys that didn't have a prefix right? Should we still redact those just in case someone uses this in a different context where it might apply to those API keys?

Reply:
Done in https://github.com/RevenueCat/purchases-ios/pull/5897/commits/816dcfa0383bdda3f02df57c07a2031de292e2e9